packages/css/src/types.ts

```diff
@@ -52,12 +52,16 @@ export type MediaQueries<StyleType> = Query<'@media', StyleType>;
 export type FeatureQueries<StyleType> = Query<'@supports', StyleType>;
 export type ContainerQueries<StyleType> = Query<'@container', StyleType>;
 export type Layers<StyleType> = Query<'@layer', StyleType>;
+export type StartingStyleQueries<StyleType> = {
```

Comment:
Thoughts on the `StartingStyleQueries` name? While the `Queries` suffix aligns with most of the other types, it doesn't really make sense since `@starting-style` isn't a query like `@media` or `@container`. It's more similar to `@layer` in that it _declares_ something. Maybe it should just be called `StartingStyle`?